### DISCUSSION TITLE
Any way to get someones coinbaseAccountId via their email?

### DISCUSSION THREAD

**Discussion by chrber04** [bennycode/coinbase-pro-node#752] [Q&A]
We were intending to use the Coinbase API for sending cryptocurrency to our users, either via their Coinbase email or a cryptocurrency address. The coinbase-node api seems to be able to accomplish both of these things, but as you all know it has been deprecated for a long time now.

That's where this library comes in as a savior, but it does not seem like it can accomplish sending cryptocurrency to another user using just their email. There is a withdrawToCoinbaseAccount(); method but as the name suggests it's to an account and not an email.

Is there any way to get someones account id using their coinbase email? Or will we be forced to use the deprecated library instead?

**Comment by bennycode** [comment]
Hi @chrber04, can you point me to the code of the deprecated library that does that? It might be simple to add this endpoint here as well.
